docs/getting-started/install/index.md

```diff
@@ -16,23 +21,51 @@ $ brew tap shopify/shopify
 $ brew install shopify-cli
 ```
 
-### apt (Debian, Ubuntu)
+**RubyGems**
+
+See the RubyGems section below for further details.
+
+### Debian/Ubuntu Linux
 
-You’ll need to install a downloaded .deb file with an explicit version number. Check the [releases page](https://github.com/Shopify/shopify-app-cli/releases) to make sure you install the latest package.
+On Debian-based Linux systems, Shopify App CLI is available via the `apt` command _or_ RubyGems.
 
+**apt**
+
+You’ll need to install a downloaded `.deb` file with an explicit version number. Check the [releases page](https://github.com/Shopify/shopify-app-cli/releases) to make sure you install the latest package.
+
+1. Download the `.deb` file from the [releases page](https://github.com/Shopify/shopify-app-cli/releases)
+1. Install the downloaded file
 ```console
 $ sudo apt install shopify-cli-x.y.z.deb
```

Comment:
We've had instances of people getting failures to do this by not using the right path to the `.deb` file. Should we make that explicit here?

Reply:
I've added `/path/to/downloaded/shopify-cli-x.y.z...`